assets/js/views/nldiView.js

```diff
@@ -4,24 +4,25 @@ import partial from 'lodash/partial';
 import NldiNavPopupView from './nldiNavPopupView';
 import * as nldiModel from '../nldiModel';
 import { getAnchorQueryValues } from '../utils';
+import axios from 'axios';
 
 /*
- * Creates the NHLD maps, an inset map and a larger map. Only one of the maps is shown.
- * The map shown is changed by clicking the expand/collapse control in the upper right of each map.
- * Each map also contains the Navigation selector.
+ * Creates the NHLD map.
+ * The map is shown/hidden by clicking the Hide/Show Upstream Downstream Mapper button.
+ * The also contains the Navigation selector.
```

Comment:
```suggestion
 * The map also contains the Navigation selector.
```
Missing the word "map"?

Reply:
Fixed in https://github.com/NWQMC/WQP_UI/pull/1019/commits/60c6b043fc6e28e1c016c9372a7539830b6c7955